src/ert/_c_wrappers/enkf/res_config.py

```diff
@@ -174,6 +177,22 @@ def _log_config_content(self, config_content: ConfigContent) -> None:
         logger.info("Content of the config_content:")
         logger.info(tmp_dict)
 
+    @staticmethod
+    def _create_predefined_defines(
```

Comment:
How does `_create_pre_defines` sound? In place of `_create_predefined_defines`.

Reply:
I'm still rooting for `predefineds`, but i'm not sure if it will stick ; )

will rename once we figure out where to move the method!

Reply:
it is done.